### PR TITLE
Unifymodify

### PR DESCRIPTION
**Pull request by massimilian-89**
Hello 
In this PR , The connectivity examples(mostly figures ,some  readme) are being updated. Please let me know if there is a need for further modifications.

### CODE REVIEW

File: connectivity/Coexistence-example/Readme.md [hunk 1]
```diff
@@ -409,6 +409,6 @@ Zooming in on Fig. 9 is clear that there is Tx/Rx activity on the 1st depicted c
 ## Known Limitations
 
 - No known hardware limitations for this example, but it is recommended to see the following:
-  - Application note for [known hardware limitations for DA14531 devices](https://www.dialog-semiconductor.com/da14531_HW_Limitation)
+  - Application note for [known hardware limitations for DA14531 devices](https://www.renesas.com/eu/en/document/apn/b-075-da14530531-hardware-guidelines?r=1564826)
```
Comment (NiekIlmer) [inline connectivity/Coexistence-example/Readme.md:412]:
```suggestion
  - Application note for [known hardware limitations for DA14531 devices](http://lpccs-docs.renesas.com/sdk6_kll/index.html)
```
This link should probably be to the KLL

File: connectivity/Coexistence-example/media/DevKit531.svg
Comment (koen-vdm) [inline connectivity/Coexistence-example/media/DevKit531.svg:1]: the text is streched on this one

File: connectivity/Coexistence-example/media/DevKit585_586.svg
Comment (koen-vdm) [inline connectivity/Coexistence-example/media/DevKit585_586.svg:1]: text is streched

File: connectivity/active_scanner/Readme.md [hunk 2]
```diff
@@ -19,19 +19,20 @@ The user can read the peripheral advertising data in a friendly way on its seria
 * **Software configuration**
 
     - This example requires:
-        * [SDK6 latest version](https://www.dialog-semiconductor.com/da14531_sdk_latest).
+        * [SDK6 latest version](https://www.renesas.com/eu/en/document/swo/sdk601811821-da1453x-da145856?r=1564826).
```
Comment (koen-vdm) [inline connectivity/active_scanner/Readme.md:22]: If you remove ?r=xxx from the url, it won't break when there is a new release.

File: connectivity/ble-Midi/Readme.md [hunk 3]
```diff
@@ -37,17 +37,17 @@ The example is running from SRAM. To run the program from flash please visit cha
...
 
 	- SW3 (button 3) is configured to P1_1, located on J19 (red box)
 	- Connect the DA145xxDEVKT-P PRO-Motherboard to the working station through USB1 connector.
 
-	![Motherboard_Hardware_Configuration_DA14585](assets/Motherboard_Hardware_Configuration_DA14585.png)
+	![Motherboard_Hardware_Configuration_DA14585](assets/Motherboard_Hardware_Configuration_DA14585.svg)
 	
 - **Software configuration**
-    - [SDK6.0.14](https://www.dialog-semiconductor.com/da14531_sdk_latest)
+    - [SDK6 latest version](https://www.renesas.com/eu/en/document/swo/sdk601811821-da1453x-da145856?r=1564826)
```
Comment (koen-vdm) [inline connectivity/ble-Midi/Readme.md:50]: If you remove ?r=xxx from the url, it won't break when there is a new release.

File: connectivity/advertising_example/assets/Motherboard_Hardware_Configuration_DA14531.svg
Comment (koen-vdm) [inline connectivity/advertising_example/assets/Motherboard_Hardware_Configuration_DA14531.svg:1]: I think you forgot to remove some pngs in this folder

File: connectivity/ble-Midi/assets/Motherboard_Hardware_Configuration_DA14531.png
Comment (koen-vdm) [inline connectivity/ble-Midi/assets/Motherboard_Hardware_Configuration_DA14531.png:1]: in the original figure there is also a box around J19

File: connectivity/ble-Midi/assets/Motherboard_Hardware_Configuration_DA14585.svg
Comment (koen-vdm) [inline connectivity/ble-Midi/assets/Motherboard_Hardware_Configuration_DA14585.svg:1]: missed a square around J19

File: connectivity/ble_pressure_sensor_bmp388/Readme.md [hunk 4]
```diff
@@ -17,12 +17,12 @@ When connected to a BLE central this example allows the measured pressure and te
 
 ### **Software configuration**
 - This example requires:
-  * [SDK6.0.14](https://www.dialog-semiconductor.com/da14531_sdk_latest).
+  * [SDK6 latest version](https://www.renesas.com/eu/en/document/swo/sdk601811821-da1453x-da145856?r=1564826).
```
Comment (koen-vdm) [inline connectivity/ble_pressure_sensor_bmp388/Readme.md:20]: remove ?r=xx

File: connectivity/ble_pressure_sensor_bmp388/Readme.md [hunk 5]
```diff
@@ -114,7 +114,7 @@ If you want to use the I2C interface, no jumpers wires are required, you can ref
 ## Known Limitations
 
 - There are no known limitations for this example. But you can check and refer to the following application note for
-[known hardware limitations for DA1458x devices](https://www.dialog-semiconductor.com/sites/default/files/da1458x-knownlimitations_2019_01_07.pdf) or [known hardware limitations for DA14531 devices](https://www.dialog-semiconductor.com/da14531_HW_Limitation).
+[known hardware limitations for DA1458x devices](https://www.renesas.com/sg/en/document/oth/da1458x-hardware-known-limitations?r=1600291) or [known hardware limitations for DA14531 devices](https://www.renesas.com/sg/en/document/dve/da14531-errata-datasheet?r=1564826).
```
Comment (koen-vdm) [inline connectivity/ble_pressure_sensor_bmp388/Readme.md:117]: remove ?r=xxx

File: connectivity/ble_pressure_sensor_bmp388/assets/PRO-DK-MikroBus-BMP388.svg
Comment (koen-vdm) [inline connectivity/ble_pressure_sensor_bmp388/assets/PRO-DK-MikroBus-BMP388.svg:1]: why is the click board in a different socket than the original picture?

File: connectivity/ble_temperature_ntf/Readme.md [hunk 6]
```diff
@@ -78,7 +78,7 @@ __Software configuration__
 
 This example requires:
 
-- [SDK6.0.14](https://www.dialog-semiconductor.com/da14531_sdk_latest).
+- [SDK6 latest version](https://www.renesas.com/eu/en/document/swo/sdk601811821-da1453x-da145856?r=1564826).
```
Comment (koen-vdm) [inline connectivity/ble_temperature_ntf/Readme.md:81]: remove ?r=xxx

File: connectivity/ble_temperature_ntf/assets/basic.svg
Comment (koen-vdm) [inline connectivity/ble_temperature_ntf/assets/basic.svg:1]: this is stull just a picture? was there no svg available?

File: connectivity/ble_temperature_ntf/assets/da14531_temp.svg
Comment (koen-vdm) [inline connectivity/ble_temperature_ntf/assets/da14531_temp.svg:1]: clickboard moved to different socket

File: connectivity/central/Readme.md [hunk 7]
```diff
@@ -34,18 +34,18 @@ The user manuals for the development kits can be found:
...
 	  **#define UART2_TX_PIN            GPIO_PIN_5**
 
-	![da14531_usb.png](assets/da14531_usb.png).
+	![da14531_usb.png](assets/da14531_usb.svg).
 	
 * **Software configuration**
 
     - This example requires:
-        * [SDK6.0.14](https://www.dialog-semiconductor.com/da14531_sdk_latest), Older versions of SDK6 require minimal changes.
+        * [SDK6 latest version](https://www.renesas.com/eu/en/document/swo/sdk601811821-da1453x-da145856?r=1564826), Older versions of SDK6 require minimal changes.
```
Comment (koen-vdm) [inline connectivity/central/Readme.md:48]: remove ?r=xxx

File: connectivity/simple_beacon/Readme.md [hunk 8]
```diff
@@ -72,12 +72,12 @@ In case of DA14531, the ***DA145xx Pro Development Kit*** is also required for t
...
 Jumpers are placed in default configurations. Refer to the getting started guide of [DA145xx Pro Development Kit](https://www.dialog-semiconductor.com/da14531-getting-started).
 
-![531_pro_dk](assets/531_pro_dk.png)
+![531_pro_dk](assets/531_pro_dk.svg)
 
 ### Software configuration
 
 
-  - [SDK6.0.14](https://www.dialog-semiconductor.com/da14531_sdk_latest) or later
+  - [SDK6.0.18](https://www.renesas.com/eu/en/document/swo/sdk601811821-da1453x-da145856?r=1564826) or later
```
Comment (koen-vdm) [inline connectivity/simple_beacon/Readme.md:80]: remove ?r=xxx

File: connectivity/switching_roles/Readme.md [hunk 9]
```diff
@@ -28,15 +28,15 @@ To run the program from flash or OTP, please visit chapter 11 of the [SmartSnipp
...
+![mb_setup](assets/mb_setup.svg)
 
 ### Software configuration
 
 This example requires:
 
 - Terminal software like Termite or Teraterm
 - BLE scanner app like BLE scanner for Android (can be found [here](https://play.google.com/store/apps/details?id=com.macdom.ble.blescanner))
-- [SDK6.0.16](https://www.dialog-semiconductor.com/da14531_sdk_latest)
+- [SDK6 latest version](https://www.renesas.com/sg/en/document/swo/sdk601811821-da1453x-da145856?r=1564826)
```
Comment (koen-vdm) [inline connectivity/switching_roles/Readme.md:39]: remove ?r=xxx

**Review by koen-vdm — CHANGES_REQUESTED** [review]
Nice improvement. some minor updates are needed

**Comment by massimilian-89** [comment]
Guy I am done here please review and give me the hints

**Review by koen-vdm — CHANGES_REQUESTED** [review]
Some issues I pointed out last review are still present. and a question is still unanswered

**Review by koen-vdm — APPROVED** [review]

**Review by NiekIlmer — APPROVED** [review]

**Pull request merged**